Ashenzari reforge (#1736)
crawl/crawl: crawl-ref/source/dat/descript/ability.txt

# Ashenzari
Curse Item ability

> Curses a chosen equipped item, binding it to the wearer until the curse
> is broken by destroying the item.

Reviewer: ```suggestion
Curses a chosen equipped item, binding it to the wearer until the curse
and item both are broken.
```
